nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutSQL.java

```diff
@@ -819,7 +819,7 @@ private void setParameter(final PreparedStatement stmt, final String attrName, f
                             bValue = DatatypeConverter.parseBase64Binary(parameterValue);
                             break;
                         default:
-                            throw new ParseException("Unable to parse binary data using the formatter `" + valueFormat + "`.",0);
+                            throw new ParseException("Unable to parse binary data using the simpleDateFormat `" + valueFormat + "`.",0);
```

Comment:
This change looks odd (no other reference to such things), perhaps a rebase is needed? Or maybe a search-and-replace ran rampant and this file should be removed from the PR?